### PR TITLE
Removed use of openPortLineBreak/LineBreakTransformer in unlockHuaweiShell()

### PR DESCRIPTION
**Pull request by ErnyTech**

### CODE REVIEW

File: assets/js/rootLantiq.js [hunk 1]
```diff
@@ -36,35 +36,34 @@ async function checkUbootUnlocked(serial) {
...
-                writer.write('f\n');
-            }, 10);
-
-            outputMsgCallback("Root in progress: Trigger characters received. Waiting for boot to end...");
-            await delay(3000);
-            clearInterval(interval);
-            break;
+async function waitFailbackShell(serial, outputMsgCallback) {
+    await serial.readLine((line) => {
+        console.log(line);
```
Comment (simonebortolin) [inline assets/js/rootLantiq.js:41]: why?

Comment (ErnyTech) [reply assets/js/rootLantiq.js:41]: Debug, this should be removed

File: assets/js/rootLantiq.js [hunk 2]
```diff
@@ -36,35 +36,34 @@ async function checkUbootUnlocked(serial) {
...
+        if (line.startsWith('Press the [f] key and hit [enter] to enter failsafe mode')) {
+            return true;
         }
-    }
+    });
 
-    const interval = setInterval(function() {
-        writer.write(String.fromCharCode(10));
+    let interval = setInterval(function() {
+        serial.writeString('f\n');
```
Comment (simonebortolin) [inline assets/js/rootLantiq.js:48]: why? `f\n`?  `String.fromCharCode(10)` is only `\n`

Comment (ErnyTech) [reply assets/js/rootLantiq.js:48]: This is the previous interval that was moved out of the loop, it hasn't been changed just moved down

**Pull request closed without merging**